ext/bootsnap/bootsnap.c

```diff
@@ -331,6 +334,32 @@ bs_rb_fetch(VALUE self, VALUE cachedir_v, VALUE path_v, VALUE handler, VALUE arg
   return bs_fetch(path, path_v, cache_path, handler, args);
 }
 
+/*
+ * Entrypoint for Bootsnap::CompileCache::Native.precompile.
+ * Similar to fetch, but it only generate the cache if missing
+ * and doesn't return the content.
```

Comment:
The `precompile` command is a bit slow on large repos/gemfiles, so I think having a dedicated `.precompile` method that does a bit less work is worth it.